inc/functions/options.php

```diff
@@ -232,7 +234,7 @@ function get_rocket_cache_reject_uri() { // phpcs:ignore WordPress.NamingConvent
 	}
 
 	// Exclude feeds.
-	$uris[] = '/(.+/)?' . $GLOBALS['wp_rewrite']->feed_base . '/?';
+	$uris[] = '/(.+/)?' . $GLOBALS['wp_rewrite']->feed_base . '/?.+/?';
```

Comment:
```suggestion
	$uris[] = '/(.+/)?' . $wp_rewrite->feed_base . '/?.+/?';
```

And add `global $wp_rewrite;` at the start of the function. This is for consistency with the rest of the code 👍